src/Angor/Client/Pages/Founder.razor

```diff
@@ -111,15 +106,16 @@ else
     {
         scanningForProjects = true;
 
+        // Clear the existing projects before rescan
```

Comment:
I agree I will remove this lines

Reply:
@dangershony @DavidGershony I added this because of the rescan feature,
now if you rescan there are duplicates.

maybe theres a better solution instead of deleting, to just add the ones that are different (we will also need to add logic for the same project being changed on another client)

![image](https://github.com/user-attachments/assets/575c430e-2a44-44b0-977d-a67a6fd24660)

Reply:
depending on your answer, i'll open a PR with the according fix